PendingReleaseNotes.md

```diff
@@ -42,6 +42,7 @@ So the CephCLuster spec field `image` must be updated to point to quay, like `im
   - Ceph v16.2.5 or greater is required for stretch clusters
 - The use of peer secret names in CephRBDMirror is deprecated. Please use CephBlockPool CR to configure peer secret names and import peers. Checkout the `mirroring` section in the CephBlockPool [spec](Documentation/ceph-pool-crd.md#spec) for more details.
 - Update Ceph CSI to `v3.4.0` for more details read the [official release note](https://github.com/ceph/ceph-csi/releases/tag/v3.4.0)
+- Enhance `CephObjectStoreUser` to include features such as quota(limitation for bucket, object, size) and RGW user capabilities.
```

Comment:
These release notes will only apply now to v1.8, so no need to update it. The v1.7.x patch releases will have changes listed based on the PR titles so we will include it no problem.